nengo_loihi/loihi_interface.py

```diff
@@ -397,6 +399,8 @@ def __init__(self, cx_model, seed=None, snip_max_spikes_per_step=50):
         if seed is not None:
             warnings.warn("Seed will be ignored when running on Loihi")
 
+        N2SpikeProbe.probeDict = {}
```

Comment:
I would lean toward doing `N2SpikeProbe.probeDict.clear()` unless there was a specific reason you went with assigning a new dict; thoughts?

Reply:
I just wanted to be super sure we were starting from a fresh dictionary that nothing else had a handle to.  But it's also true that if other things _do_ have a handle to the old (broken) probe dict, then `.clear` might fix any issues for them as well.  I'm fine either way :+1: